tests/server/conftest.py

```diff
@@ -128,6 +129,16 @@ def inner(
         else:
             assert expected_ids == response_ids
 
+        expected_warnings = expected_warnings or []
```

Comment:
What is the purpose of this line? `expected_warnings = expected_warnings or []`
for the subsequent if statement it does not matter whether 'expected warnings' is 'None' or '[]'.

Reply:
True, this is now redundant as the `else` branch does not test for the length of the warnings anymore.

Reply:
```suggestion
```